generated-platform-project/quarkus-google-cloud-services/bom/pom.xml

```diff
@@ -2667,12 +2667,12 @@
       <dependency>
         <groupId>com.google.api</groupId>
         <artifactId>gax-grpc</artifactId>
-        <version>2.24.0</version>
+        <version>2.16.0</version>
```

Comment:
@aloubyansky this change looks a bit suspicious. Looks like the version is aligned with the gRPC API version AFAICS so maybe it's for the better, I dunno.

Reply:
Right, it seems to be aligned with
```
<dependency>
  <groupId>com.google.api.grpc</groupId>
  <artifactId>proto-google-common-protos</artifactId>
  <version>2.16.0</version>
</dependency>
```
enforced in the quarkus-bom.